content/ui-patterns/navigation.mdx

```diff
@@ -0,0 +1,392 @@
+
+On narrow viewports, the parent list may be stacked above the detail view or turned into an action menu. See the [responsive layout guidelines](/foundations/layout#page-types) for more info.
+
+#### Tabbed interfaces
+
+<img
+    width="960"
+    alt="Screenshot of a repo's security advisories page with the tabs and tab panel areas highlighted"
+    src="https://github.com/primer/design/assets/2313998/f7fb7251-f9bf-4d58-b6ec-16b66f3959da"
+/>
+
+Tabbed interfaces enable seamless navigation between different subpages within a unified context. Tabs go directly above the content they affect.
+
+### Headings and hierarchical page sections
+
+<img
+    width="960"
+    alt="Wireframe of a settings page with multiple nested sections"
+    src="https://github.com/primer/design/assets/2313998/88dedda4-355c-48ba-af14-9c01934dd276"
+/>
+
+A [page header](/components/page-header) is the title area at the top of the main content region that shows the user where they are. It may also to contain navigational elements like breadcrumbs or a link to go up a level of hierarchy.
+
+Use section headings to show the user where they are in the context of the page. These headings should be styled to reflect their hierarchy within the page. Page section headings should be marked up with the appropriate heading level (`h3`–`h6`).
+
+See our [typographic style options](/primitives/storybook/?path=/story/typography-functional--overview) for heading styles.
+
+
+## Accessibility
+
+### Skip links
+
+Facilitate efficient navigation for screen reader users by including skip links to bypass dense redundant content and navigate directly to essential sections.
+
+Example for skipping redundant content: there is a skip link near the global navigation that allows users skip over all the global navigation items and go straight to the main content.
+
+Example for skipping dense content: there is a skip link near the header of a data table that allows users to skip over the large amount of content often rendered in a data table.
+
+### Tab order
+
+Users should be able to tab through the navigation elements in a logical order. The tab order should reflect the visual order of the elements.
+
+### Scannable headings
+
+A correct heading structure is critical for enabling users to navigate quickly within a page. Headings are by far the most common navigation technique for screen reader users.
+
+Follow our accessibility [guidelines for headings](/guides/accessibility/headings).
+
+
+## Related links
+
+- [Action menu](/components/action-menu)
+- [Breadcrumbs](/components/breadcrumbs)
+<!-- markdownlint-disable GH002 "Avoid using generic link text" - "Link" is actually the name of the component -->
+- [Link](/components/link)
+- [Pagination](/components/pagination)
+- [Segmented control](/components/segmented-control)
+- [Underline nav](/components/underline-nav)
+- [Underline panels](/components/underline-panels)
+
```

Comment:
Did you mean to have a space here? It looks kinda funny in the deploy but not sure..

<img width="487" alt="related links section" src="https://github.com/primer/design/assets/18661030/97363d6f-85f7-43b7-91d5-55e3c0fa3f06">

Reply:
Oops - that's just a rogue return. Will remove.